src/nft/hooks/useNFTSelect.ts

```diff
@@ -25,7 +25,7 @@ export const useNFTSelect = create<SelectNFTState>()(
       selectNFT: (nft) =>
         set(({ selectedNFTs }) => {
           if (selectedNFTs.length === 0) return { selectedNFTs: [nft] }
-          else if (!!selectedNFTs.find((x) => x.id === nft.id))
+          else if (selectedNFTs.find((x) => x.id === nft.id))
```

Comment:
`find` is incorrect - use [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)